src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  // Dividir a string em declarações separadas por ';'
+  const declarations = sourceString.split(';');
```

Comment:
Consider trimming the `sourceString` before splitting it to handle any leading or trailing semicolons that might result in empty declarations.